_posts/en/newsletters/2019-10-09-newsletter.md

```diff
@@ -0,0 +1,171 @@
+---
+title: 'Bitcoin Optech Newsletter #67'
+permalink: /en/newsletters/2019/10/09/
+name: 2019-10-09-newsletter
+slug: 2019-10-09-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter requests help testing release candidates for
+Bitcoin Core and LND, tracks continued discussion about the proposed
+noinput and anyprevout sighash flags, and describes several notable
+changes to popular Bitcoin infrastructure projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Help test Bitcoin Core 0.19.0RC1:** production users of Bitcoin Core
```

Comment:
FWIW the release is tagged as `v0.19.0rc1      Bitcoin Core 0.19.0 release candidate 1` (`git tag -n | sort -V`)